Libraries/CO.CDP.AwsServices/CloudWatchExtensions.cs

```diff
@@ -35,9 +35,9 @@ private static IServiceCollection AddCloudWatchSinkOptions(this IServiceCollecti
             // options for the sink defaults in https://github.com/Cimpress-MCP/serilog-sinks-awscloudwatch/blob/master/src/Serilog.Sinks.AwsCloudWatch/CloudWatchSinkOptions.cs
             return new CloudWatchSinkOptions
             {
-                LogGroupName = awsConfiguration.LogGroup,
+                LogGroupName = awsConfiguration.CloudWatch.LogGroup,
```

Comment:
Once LogGroup is made required non-null, `awsConfiguration?.CloudWatch.LogGroup`